documentation/add/09_test_data_storage.md

```diff
@@ -0,0 +1,94 @@
+# Test data storage, distribution and requirements for PACE project
+
+Date: 3-11-2021
+
+## Current State
+Currently, the test data are stored on the GitHub Repo as part of the Herbert/Horace project. 
+Anyone who clones the GitHub repo also clones all test data and the history of binary mat files.
+
+All users of the GitHub version have direct access to the test data, stored in the correct location.
```

Comment:
Sorry, I will clarify. By that, I mean that the data are in the location where they are used (i.e. the correct search path) rather than having to manually be copied/moved to the correct locations.